data/alpha/register/social-housing-provider-designation.yaml

```diff
@@ -0,0 +1,9 @@
+fields:
+- social-housing-provider-designation
+- name
+- start-date
+- end-date
+phase: alpha
+register: social-housing-provider-designation
+registry: homes-and-communities-agency
+text: The Designation of a social housing provider
```

Comment:
register descriptions have no full stop field descriptions do. having a full stop on the register descriptions cause the tests to fail